osu.Game.Rulesets.Taiko/Mods/TaikoModEasy.cs

```diff
@@ -7,6 +7,6 @@ namespace osu.Game.Rulesets.Taiko.Mods
 {
     public class TaikoModEasy : ModEasy
     {
-        public override string Description => @"Beats move slower, less accuracy required, and three lives!";
+        public override string Description => @"Beats move slower, less accuracy required!";
```

Comment:
Still no "and".
```suggestion
        public override string Description => @"Beats move slower, and less accuracy required!";
```

Reply:
yeah, figured i'd leave it as is at that point, but sure